Code-4-Community/lucys-love-bus-backend
# Add privilegeLevel field to UserInformation class and methods returning UserInformation

## Why

When an admin goes to approve/deny a PF request, they shouldn't be able to overwrite any past approvals/denials. To solve this, we are using the user's privilege level to see if their PF request has been dealt with. Therefore we need to return the privilege level of the user along with the rest of the UserInformation

## This PR

- added the privilege level field to the UserInformation class
- returned the user's privilege level in the getRequestData method

## Screenshots

N/A

## Verification Steps

- create a PF request through the UI by signing up as a PF 
- make the GET api/v1/protected/requests/:request_id/:user_id method with the corresponding request id and user id (find them in your local DB) 
- see the privilege level field being included in the response (can be done in the browser or using Postman)
